Userland/Libraries/LibJS/Runtime/Intrinsics.h

```diff
@@ -0,0 +1,163 @@
+/*
```

Comment:
Seems you forgot to add this file to CMakeLists.txt (or push it) Since lagom just globs for cpp files

Reply:
(This was meant on the cpp file of course)

Reply:
every damn time lol

...thanks, fixed!